e2e/provision/gce_install_sandbox.sh

```diff
@@ -22,8 +22,7 @@ function deploy_kpt_pkg {
   local temp=$(mktemp -d -t kpt-XXXX)
   local localpkg="$temp/$name"
   kpt pkg get --for-deployment "https://github.com/nephio-project/nephio-example-packages.git/$pkg" "$localpkg"
-  # sudo because docker
-  sudo kpt fn render "$localpkg"
+  kpt fn render "$localpkg"
```

Comment:
Another way to achieve that is using `newgrp`
```suggestion
  newgrp docker <<EONG
  kpt fn render "$localpkg"
EONG
```

Reply:
OK, newgrp spawns a new shell and then the render happens in that shell with the new privileges.

Reply:
Hi @electrocucaracha , I am currently working on refactoring the scripts a bit so will take the docker group issue after this change if that's ok.
Plan is to add an intermediary script to setup the platform and isolate and sudo calls.

Reply:
Hi @electrocucaracha I have reverted this and left in the sudo.